hail/src/test/scala/is/hail/annotations/StagedRegionValueSuite.scala

```diff
@@ -231,6 +231,47 @@ class StagedRegionValueSuite extends SparkSuite {
       new UnsafeIndexedSeq(rt.physicalType, rv2.region, rv2.offset)))
   }
 
+  @Test
+  def testMissingRandomAccessArray() {
+    val rt = TArray(TStruct("a" -> TInt32(), "b" -> TString()))
```

Comment:
Can you also add the struct case?

Reply:
I will make sure that the new functions have a test case, but there's no current 'set all missing then randomly set fields' pattern for structs.